### PR TITLE
Upgrading core

### PR DESCRIPTION
**Pull request by kneth**
@realm/java

### CODE REVIEW

**Comment by cmelchior** [comment]
👍

**Comment by stk1m1** [comment]
👍  as soon as CI passes.

**Comment by nhachicha** [comment]
👍

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,11 @@
+## 0.91.1
+
+* Updated Realm Core.
```
Comment (dalinaum) [inline CHANGELOG.md]: `Update Realm Core to 1.0.1.`?

**Comment by dalinaum** [comment]
Besides a minor comment, 👍

**Comment by kneth** [comment]
retest this please

**Pull request merged**